core/templates/pages/library-page/search-bar/search-bar.directive.html

```diff
@@ -126,12 +126,12 @@
     padding-right: 0;
   }
   .oppia-search-bar-input.btn {
-    color: rgba(255,255,255,0.7);
+    color: rgba(255,255,255,.95);
```

Comment:
Why these changes?

Reply:
These are color contrast changes to the community library. You can see it in the screen shots.

Reply:
optional: Use `0.95` for consistency